docs/DeveloperGuide.md

```diff
@@ -154,6 +154,28 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Lead Status feature
+
+The Lead Status feature aims to provide information about the contact based on when the status was last set.
+It is represented by the `LeadStatus status` attribute in a `Person`, which contains a `LeadStatusName` and 
+`LocalDateTime` timestamp.
```

Comment:
`
### Lead Status feature

#### About
`


I've added a sub heading "about" for my feature description!
Perhaps can here too for consistency.